p2p/sentry/sentry_grpc_server.go

```diff
@@ -646,27 +647,30 @@ func NewGrpcServer(ctx context.Context, dialCandidates func() enode.Iterator, re
 				// handshake is successful
 				logger.Trace("[p2p] Received status message OK", "peerId", printablePeerID, "name", peer.Name())
 
-				ss.GoodPeers.Store(peerID, peerInfo)
-				ss.sendNewPeerToClients(gointerfaces.ConvertHashToH512(peerID))
 				getBlockHeadersErr := ss.getBlockHeaders(ctx, *peerBestHash, peerID)
```

Comment:
This call doesn't belong here. I don't understand why is it needed, we should investigate using git history. In the long run it should be extracted out of this class together with the handling code (if we find it). This can be a separate task that we do in the future.

In any case, I think that this call should happen after runPeer in parallel with it. In this PR I'd like to either:
* start it in a separate goroutine after runPeer() call (this would be a more "correct" init sequence)
* or keep it after `ss.sendNewPeerToClients` and before `return runPeer` as it was before (this would be the old unchanged init sequence)

Reply:
ok ill revert these changes